src/relayclient/GSNConfigurator.ts

```diff
@@ -89,8 +86,10 @@ export async function resolveConfigurationGSN (provider: Web3Provider, partialCo
   ] = await Promise.all([
 
     partialConfig.chainId ?? contractInteractor.getAsyncChainId(),
-    paymasterInstance.getHubAddr().catch(e => { throw new Error('Not a paymaster contract') }),
-    partialConfig.forwarderAddress ?? paymasterInstance.trustedForwarder().catch(e => { throw new Error('paymaster has no trustedForwarder()') }),
+    // @ts-ignore
```

Comment:
Why did you need a `ts-ignore` here?

Reply:
I think the "as Error" resolved it (and that's why ts-error is so much better)